legal-api/src/legal_api/services/search_service.py

```diff
@@ -269,3 +269,32 @@ def get_search_filtered_filings_results(business_json,
                 draft_results.append(draft)
 
         return draft_results
+
+    @staticmethod
+    def get_affiliation_mapping_results(org_id: int = None):
+        """Return affiliation mapping results for the given organization ID."""
+        query = db.session.query(
+            Filing.id.label('filing_id'),
+            Business.identifier,
+            Filing
+            .filing_json['filing'][Filing._filing_type]['nameRequest']['nrNumber']  # pylint: disable=protected-access
+            .label('nrNum'),
+            RegistrationBootstrap.identifier.label('temp_reg'),  # pylint: disable=no-member
+            RegistrationBootstrap.account.label('account_id')
+                ).select_from(Filing) \
+            .outerjoin(Business, Filing.business_id == Business.id) \
+            .join(RegistrationBootstrap, Filing.temp_reg == RegistrationBootstrap.identifier)  \
```

Comment:
for migrated corps, we don't have a registration bootstrap entry which will mean these corps will get excluded on this join.  @ochiu are you planning on using this endpoint?

Reply:
yeah, org id is probably problematic but just filtering by identifier still misses draft scenarios as there won't be an identifier yet.  does this endpoint need to support the draft scenarios?

Reply:
yes, LEAR will have the temp identifier.  This endpoint will just need to support querying by the business identifier and temp identifier

Reply:
here's an example payload:

`identifiers: ['NR393435', 'TW322340', 'BC1234567']`